Source/WebCore/page/LocalFrameView.cpp

```diff
@@ -4772,6 +4772,8 @@ void LocalFrameView::setWasScrolledByUser(bool wasScrolledByUser)
     cancelScheduledScrolls();
     if (currentScrollType() == ScrollType::Programmatic)
         return;
+    if (wasScrolledByUser)
```

Comment:
Won't the indicator still flash on every stylesheet load with this change?

Reply:
No, it will only flash when it's loaded the first time, not after the style sheet loads. We do still want it to be re-centered, but we don't want it to re-flash.